Breeder/BR_MouseUtil.cpp

```diff
@@ -80,26 +85,30 @@ static MediaTrack* GetTrackAreaFromY (int y, int* offset)
 			break;
 		}
 		trackOffset += height;
+		trackSpacerHeight = 0;
 	}
 
 	WritePtr(offset, (track) ? (trackOffset) : (0));
+	WritePtr(spacerHeight, trackSpacerHeight);
 	return track;
 }
 
-static MediaTrack* GetTrackFromY (int y, int* trackHeight, int* offset)
+static MediaTrack* GetTrackFromY (int y, int* trackHeight, int* offset, int* spacerHeight)
 {
 	int trackOffset = 0;
 	int trackH = 0;
-	MediaTrack* track = GetTrackAreaFromY(y, &trackOffset);
+	int spacer = 0;
+	MediaTrack* track = GetTrackAreaFromY(y, &trackOffset, &spacer);
 	if (track)
 	{
 		trackH = GetTrackHeight(track, NULL);
 		if (y < trackOffset || y >= trackOffset + trackH)
 			track = NULL;
 	}
 
-	WritePtr(trackHeight, (track) ? (trackH)      : (0));
-	WritePtr(offset,      (track) ? (trackOffset) : (0));
+	WritePtr(trackHeight,  (track) ? (trackH)           : (0));
+	WritePtr(offset,       (track) ? (trackOffset)      : (0));
+	WritePtr(spacerHeight, (track && spacer) ? (spacer) : (0));
```

Comment:
Opportunity to cleanup the old code:

```suggestion
	WritePtr(trackHeight,  track ? trackH             : 0);
	WritePtr(offset,       track ? trackOffset        : 0);
	WritePtr(spacerHeight, (track && spacer) ? spacer : 0);
```
Last parenthesis is also unnecessary but I guess it's fine for better readability. 🤷

Reply:
👍 I made these changes and reverted them because I didn't want to fix it everywhere, but I guess local gardens are nice, too.